brian2/devices/cpp_standalone/device.py

```diff
@@ -199,6 +199,12 @@ def __init__(self):
                            'before_end': [],
                            'after_end': []}
 
+        #: Dictionary storing compile and binary execution times
+        self.timers = {'run_binary': None,
+                       'compile': {'clean':None,
```

Comment:
```suggestion
                       'compile': {'clean': None,
```
I see you did not use `black` here :wink:

Reply:
I'm not using black yet because it would reformat all of Brian's code base and this commit would have been a bit of a mess :smile:. But you know, if there was some automatic code formatting like black in place, this would have not happened 😜